pkg/kubelet/cm/dra/plugin/plugins_store.go

```diff
@@ -43,26 +42,31 @@ func (s *pluginsStore) get(pluginName string) *plugin {
 
 // Set lets you save a DRA Plugin to the list and give it a specific name.
 // This method is protected by a mutex.
-func (s *pluginsStore) add(pluginName string, p *plugin) {
+func (s *pluginsStore) add(pluginName string, p *Plugin) (replaced bool) {
 	s.Lock()
 	defer s.Unlock()
 
 	if s.store == nil {
-		s.store = make(map[string]*plugin)
+		s.store = make(map[string]*Plugin)
 	}
 
 	_, exists := s.store[pluginName]
-	if exists {
-		klog.V(1).InfoS(log("plugin: %s already registered, previous plugin will be overridden", pluginName))
-	}
 	s.store[pluginName] = p
+	return exists
```

Comment:
Why is it not sufficient to log here? Why introduce this (somewhat awkward) return value?